servers/visual_server.cpp

```diff
@@ -2401,6 +2401,7 @@ VisualServer::VisualServer() {
 	GLOBAL_DEF("rendering/quality/shading/force_lambert_over_burley.mobile", true);
 	GLOBAL_DEF("rendering/quality/shading/force_blinn_over_ggx", false);
 	GLOBAL_DEF("rendering/quality/shading/force_blinn_over_ggx.mobile", true);
+	GLOBAL_DEF("rendering/quality/shading/gles2_high_float_precision.mobile", false);
```

Comment:
Should have both the normal and the ``.mobile`` variant here.

Reply:
It's a bit of a weird situation though as the non-`.mobile` variant (or `.Android` if we go this route) wouldn't do anything currently.